src/cmd.zig

```diff
@@ -1200,17 +1206,26 @@ fn validator(
             sig.replay.service.run,
             .{sig.replay.service.ReplayDependencies{
                 .allocator = allocator,
-                .logger = app_base.logger.unscoped(),
-                .my_identity = .{ .data = app_base.my_keypair.public_key.bytes },
+                .logger = .from(app_base.logger),
+                .my_identity = .fromPublicKey(&app_base.my_keypair.public_key),
+                .vote_identity = .fromPublicKey(&app_base.my_keypair.public_key), // TODO: is this fine, or do we need a separate identity for the vote account?
```

Comment:
I think there is the need to have separate keys for the identity and voting. Although in practice the operator may choose to use the same keys. According to https://github.com/anza-xyz/agave/blob/07c4a88cdbac5dc39010edea022ec3c724214d1e/docs/src/operations/guides/vote-accounts.md?plain=1#L74-L75

Reply:
On second thought, I think this should be resolved, either by updating the code, or confirming this is fine.

Reason being that, if it is not fine, then this is probably introducing an issue. An operator may set up a vote identity with the expectation this would be used to sign the transaction, but the validator identity will be used instead.

I'll dismiss my approval based on the above.

Reply:
Given that the support of two keys is listed in the documentation of the only Solana client, and hence folks who may want to try out Sig would be familiar with that setup, could it be a point of confusion/contention? 

If not then I think, indeed it can be classified as a nice to have.

Reply:
It is okay to have the keys separated as it is now, but only accept one via the cli. Later we can extend the cli to support passing different keys for voting/identity